packages/notifier/src/notifier.js

```diff
@@ -50,94 +49,21 @@ export const makeNotifier = sharableInternalsP => {
  * @returns {NotifierRecord<T>} the notifier and updater
  */
 export const makeNotifierKit = (...initialStateArr) => {
-  /** @type {PromiseRecord<UpdateRecord<T>>|undefined} */
-  let optNextPromiseKit;
-  /** @type {UpdateCount & (number | undefined)} */
-  let currentUpdateCount = 1; // avoid falsy numbers
-  /** @type {UpdateRecord<T>|undefined} */
-  let currentResponse;
-
-  const hasState = () => currentResponse !== undefined;
+  /** @type {PublishKit<T>} */
+  const { publisher, subscriber } = makePublishKit();
 
-  const final = () => currentUpdateCount === undefined;
-
-  const baseNotifier = Far('baseNotifier', {
-    // NaN matches nothing
-    getUpdateSince(updateCount = NaN) {
-      if (
-        hasState() &&
-        (final() ||
-          (currentResponse && currentResponse.updateCount !== updateCount))
-      ) {
-        // If hasState() and either it is final() or it is
-        // not the state of updateCount, return the current state.
-        assert(currentResponse !== undefined);
-        return Promise.resolve(currentResponse);
-      }
-      // otherwise return a promise for the next state.
-      if (!optNextPromiseKit) {
-        optNextPromiseKit = makePromiseKit();
-      }
-      return optNextPromiseKit.promise;
-    },
-  });
+  // eslint-disable-next-line no-use-before-define
+  const baseNotifier = makeNotifierFromSubscriber(subscriber);
 
   const notifier = Far('notifier', {
     ...makeNotifier(baseNotifier),
     ...baseNotifier,
   });
```

Comment:
This `makeNotifier` seems to be unnecessary; everything it provides is already on the output from `makeNotifierFromSubscriber` and thus overridden by the spread.
```suggestion
  const notifier = makeNotifierFromSubscriber(subscriber);
```